### DISCUSSION TITLE
Should I enable both `enablePeriodicRefresh` and `enableAdaptiveRefreshTrigger`?

### DISCUSSION THREAD

**Discussion by tokuhirom** [redis/lettuce#2494] [Q&A] [answered]
If I set the `enableAdaptiveRefreshTrigger` as true, `enablePeriodicRefresh` is meaningless?
Or, something good thing to enable `enablePeriodicRefresh`?

**Comment by mp911de** [comment, marked as answer]
`enablePeriodicRefresh` can update the topology even if your application doesn't have any Redis interaction.

**Reply by tokuhirom** [reply]
I got it, thanks!
